packages/blocks-editor/src/controls/object/array-control.tsx

```
@@ -23,186 +35,149 @@ export const ImageArrayControl = makeArrayControl<ImageType>({
+}
+
+const ArrayControl = <vT, pT extends object = object>(
+	props: ArrayControlType<vT, pT> & ArrayControlArgs<vT>
+) =>
+	isArrayPathControl<vT, pT>(props) ? (
+		<ArrayControlByPath {...props} />
+	) : (
+		<ArrayControlByValue {...props} />
+	)
+
+function ArrayControlByPath<pT extends object, vT>({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ArrayControlByPath'. Consider to delete it or rename it.

'ArrayControlByPath' is defined here:


(lint/suspicious/noRedeclare)